tsconfig.json

```diff
@@ -3,12 +3,11 @@
 		"declaration": true,
 		"declarationMap": true,
 		"esModuleInterop": true,
-		"module": "ESNext",
-		"moduleResolution": "Bundler",
+		"module": "esnext",
+		"moduleResolution": "bundler",
 		"noEmit": true,
 		"resolveJsonModule": true,
 		"skipLibCheck": true,
-		"sourceMap": true,
```

Comment:
Marking this PR as a `feat:` because it removes source maps. I'd removed them from the template to keep package sizes small.